release notes/v0.44.0.md

```diff
@@ -0,0 +1,221 @@
+
+- `locator.click` is now asynchronous, which is a breaking change.
+- `browserContext.addCookies` has now been implemented.
+- `browserType.Connect` has been implemented so k6 can now connect to an already running Chrome/Chromium browser instance.
+- Web vitals are natively supported when working with the browser module.
+
+## Breaking changes
+
+- [browser#790](https://github.com/grafana/xk6-browser/pull/790) Converted `locator.click` to async to have feature parity with `page.click` and `elementHandle.click`. Users must remember to work with `promise.All` and `page.waitForNavigation()` when a click action results in navigation.
+
+    A `locator.click` action that doesn't result in navigation can be used like so:
+    ```javascript
+    const tails = page.locator("input[value='Bet on tails!']");
+    await tails.click(),
+    ```
+
+    A `locator.click` action that does result in a navigation can be used like so:
+    ```javascript
+    const tails = page.locator("input[value='Bet on tails!']");
+    await Promise.all([
+      page.waitForNavigation(),
+      tails.click(),
+    ]);
+    ```
+
+- [browser#817](https://github.com/grafana/xk6-browser/pull/817) We've removed `--no-sandbox` from the default Chrome launch arguments. Now Chrome will launch with a sandbox, which is a more secure way of running the browser. If you are running tests under a `root` user, the browser will no longer launch unless the `--no-sandbox` argument is supplied. You can still pass this flag when launching a new Chrome instance using the `args` parameter on `chromium.launch`:
+
+    ```javascript
+    const browser = chromium.launch({
+      args: ['no-sandbox'],
+    });
+    ```
+
+- [browser#844](https://github.com/grafana/xk6-browser/pull/844) Removed the exported `version` param from the root module. Users should from now on reference the k6 version instead of the browser module version.
+- [browser#838](https://github.com/grafana/xk6-browser/pull/838) Removed the first meaningful paint metric. This metric is being deprecated across all the browsers, because the metric's definition relies on browser-specific implementation details, and we've now introduced web vitals in the browser module which is a reliable industry standard way to measure frontend performance. You can find more details [here](https://developer.chrome.com/en/docs/lighthouse/performance/first-meaningful-paint/).
+- [browser#843](https://github.com/grafana/xk6-browser/pull/843) Removed the build step from Github Actions. From this release onwards, no new standalone browser binaries will be built and available from the [releases](https://github.com/grafana/xk6-browser/releases) section. The latest version of the browser module will be available in the k6 binary which can be found in the [k6 releases](https://github.com/grafana/k6/releases) page.
+
+## New features
+
+### A new experimental module implementing the Web Crypto API specification [#3007](https://github.com/grafana/k6/pull/3007)
```

Comment:
```suggestion
### A new `experimental/webcrypto` module implementing the Web Crypto API specification [#3007](https://github.com/grafana/k6/pull/3007)
```

Reply:
If we are going to give it as a module name then let's pretend k6/ as well

Reply:
@codebien @oleiade what do you think?

(Asking you since there was [another tread](https://github.com/grafana/k6/pull/3024#discussion_r1172364742) about the naming)

Reply:
I'm for using the current version in the title and using the suggestion in the next sentence as suggested in https://github.com/grafana/k6/pull/3024#discussion_r1174558164 but using the complete `k6/experimental/webcrypto` form.

Reply:
I'd go with @sniku 's suggestion, but use the full module path indeed 👍🏻 `k6/experimental/webcrypto` 🙇🏻